swift_browser_ui/api.py

```diff
@@ -613,43 +548,42 @@ async def get_project_metadata(
     ret = dict()
     try:
         session = api_check(request)
-        request.app['Log'].info(
-            'Api call for project metadata check from {0}, sess: {1}'.format(
+        request.app["Log"].info(
+            "Api call for project metadata check from {0}, sess: {1}".format(
                 request.remote,
                 session,
             )
         )
 
-        conn = request.app['Sessions'][session]['ST_conn']
+        conn = request.app["Sessions"][session]["ST_conn"]
 
         # Get the account metadata listing
-        stat = dict(conn.stat()['items'])
+        stat = dict(conn.stat()["items"])
         ret = {
-            'Account': stat['Account'],
-            'Containers': stat['Containers'],
-            'Objects': stat['Objects'],
-            'Bytes': stat['Bytes'],
+            "Account": stat["Account"],
+            "Containers": stat["Containers"],
+            "Objects": stat["Objects"],
+            "Bytes": stat["Bytes"],
         }
         return aiohttp.web.json_response(ret)
     except SwiftError:
-        request.app['Log'].error("SwiftError occured.")
+        request.app["Log"].error("SwiftError occured.")
         return aiohttp.web.json_response(ret)
     except ClientException as e:
-        request.app['Log'].error(e.msg)
+        request.app["Log"].error(e.msg)
         return aiohttp.web.json_response(ret)
     except KeyError:
-        request.app['Log'].error("items is missing; possible allas "
-                                 "is not authorised for project.")
+        request.app["Log"].error(
+            "items is missing; possible allas " "is not authorised for project."
```

Comment:
```suggestion
            "items is missing; possible swift storage is not authorised for project."
```
let us keep the `Allas` name out of this app